site/docs/getting-started.mdx

```diff
@@ -4,8 +4,102 @@ sidebar_position: 1
 
 # Getting Started
 
-Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+## Developing
 
-## Lorem ipsum
+Learn how to use Odyssey's [UI components](/docs/components/) in your React web applications.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+### Pre-requisites
+
+In order to use our components you will need:
+
+- Familiarity with web development fundamentals (HTML, CSS & JavaScript) and [React](https://reactjs.org/)
+- [Node.js](https://nodejs.org/) v16 or later installed on your system.
+- A React (v16 or later) web application project.
+  - For new projects, you can use [`create-react-app`](https://github.com/facebook/create-react-app#creating-an-app) or similar tools.
+
+### Install packages
+
+You will need to add the following three dependencies to your project:
+
+```sh
+npm install --save @jpmorganchase/uitk-core @jpmorganchase/uitk-theme @jpmorganchase/uitk-icons
+```
+
+:::note
+
+If you prefer, you can use [`yarn`](https://yarnpkg.com/) or other package managers to install our NPM packages.
+
+:::
+
+- `@jpmorganchase/uitk-core` contains our production-ready UI components
+- `@jpmorganchase/uitk-theme` contains the CSS files that apply Odyssey's default theme
+- `@jpmorganchase/uitk-icons` contains our SVG-based icons
+
+:::tip
+
+We strongly recommend that you regularly update to newer, compatible versions of Odyssey as they are released. For example, if using `npm`, this can be achieved by running:
+
+```sh
+npm update @jpmorganchase/uitk-core @jpmorganchase/uitk-theme @jpmorganchase/uitk-icons
+```
+
+This will provide you with the latest bug fixes and enhancements and does not require you to update any of your own code.
+
+Additionally, we recommend that you periodically check for new major versions (for example, via `npm outdated @jpmorganchase`) and aim to upgrade to those as soon as possible. Please note that new major versions will contain breaking changes, so always review the associated release notes and update your code as needed.
```

Comment:
We should have a page/section about semver usage

Reply:
I'll remove this whole box for the time being. But, yeah, in due course we should elaborate more on this. Perhaps there should be a page dedicated to upgrading that explains why / how / when to do it, since that topic hasn't always been well understood by consumers of our internal DS.